docs/instance_groups.md

```diff
@@ -12,6 +12,15 @@ By default, a cluster has:
   because we need to force the cloud to run an instance in every zone, so we can mount the master volumes - we
   cannot do that across zones.
 
+## Instance Groups Disclaimer
+
+* When there is only one availability zone in a region (eu-central-1) and you would like to run multiple masters, 
+  you have to define multiple instance groups for each of those masters. (e.g. `master-eu-central-1-a` and 
+  `master-eu-central-1-b` and so on...)
+* If instance groups are not defined correctly (particularly when there are an even number of master or multiple 
+  groups of masters into one availability zone in a single region), etcd servers will not start and master nodes will not check in. This is because etcd servers are configured per availability zone. DNS and Route53 would be the first places to check when these problems are happening.
```

Comment:
I've read this a few times now and the "etcd servers are configured per availability zone" section is somewhat confusing. The auto-generated kops config when you have more than 1 master per AZ will just name the masters with and appended number, such as `master-us-east-1a-1` and `master-us-east-1a-2`. So the etcd servers are not really configure per AZ but per master.

Please correct me if I'm misreading this.

Reply:
Please correct me if I am wrong, but in the first paragraph it states that: `you have to define multiple instance groups for each of those masters.` which agrees with your point.

Now, the second part `etcd servers are configured per availability zone` is true as well because kops will configure an instance group only based on how many availability zones are present. That's why it is important to point this out because by default, kops will configure only one instance group for the masters.

Users need to know this in order to request multiple masters in a single availability zone or an odd number of masters in an even number of availability zones... Since if there are two availability zones, then the automatically configured instance groups of 2 masters will not be happy and will fight for the quorum.